features.js

```diff
@@ -737,6 +737,31 @@ module.exports = {
         return peerConnectionLog.filter(entry => entry.type === 'oniceconnectionstatechange' && (entry.value === 'connected' || entry.value === 'completed')).length > 0;
     },
 
+    // did ICE disconnect?
+    ICEDisconnected: function(client, peerConnectionLog) {
+        return !!(peerConnectionLog.find(entry => entry.type === 'oniceconnectionstatechange' && entry.value === 'disconnected'));
+    },
+
+    // did ICE reconnect?
+    ICEReconnected: function(client, peerConnectionLog) {
+        var i = 0;
+        var connected = false;
+        for (; i < peerConnectionLog.length; i++) {
+            if (peerConnectionLog[i].type === 'oniceconnectionstatechange' && peerConnectionLog[i].value === 'disconnected') {
+                connected = true;
```

Comment:
shouldn't this variable be called `disconnected` instead of `connected` for better readability?

Reply:
copy-paste... will fix